queries.sql

```diff
@@ -43,3 +43,49 @@ SELECT  species, MIN(escape_attempts) FROM animals GROUP BY species;
 SELECT  species, MIN(weigth_kg) FROM animals GROUP BY species;
 SELECT  species, MIN(weigth_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+/*Quaries transaction day 3*/
+
+vet_clinic=# UPDATE ANIMALS SET species_id = 2 WHERE name LIKE '%mon';
+UPDATE ANIMALS SET species_id = 1 WHERE species_id <> 2;
+
+/*Quaries using JOIN */
+
+SELECT name 
+FROM animals 
+INNER JOIN owners ON owners.full_name = 'Melody Pond' 
+AND owners.id = animals.owner_id; /*What animals belong to Melody Pond? */
+
+SELECT animals.name 
+FROM animals 
+JOIN species ON  species.name = 'Pokemon'; /*List of all animals that are pokemon*/
+
+SELECT owners.full_name, animals.name  
+FROM owners 
+LEFT JOIN animals ON  owners.id = animals.owner_id;/*List all owners and their animals*/
+
+SELECT species.name, COUNT(animals.name)  
+FROM species 
+LEFT JOIN animals ON species.id = animals.species_id GROUP BY species.name; /*How many animals are there per species?*/
+
+SELECT name AS Jennifer_Animals 
+FROM animals 
+JOIN owners ON owners.id = animals.owner_id 
+AND owners.full_name = 'Jennifer Orwell';/*List all Digimon owned by Jennifer Orwell*/
+
+SELECT animals.name 
+FROM animals 
+JOIN owners ON owners.id = animals.owner_id 
+AND owners.full_name = 'Dean Winchester' 
+AND animals.escape_attempts = 0; /*List all animals owned by Dean Winchester that haven't tried to escape*/
```

Comment:
- [ ] I think there is a small issue you added two `AND`, It should be `WHERE` rather than `AND`
```
 WHERE owners.full_name = 'Dean Winchester'
 AND animals.escape_attempts = 0;
```
Kindly fix this small issue 👍